1.10.2/src/main/java/fr/ourten/brokkgui/wrapper/GuiRenderer.java

```diff
@@ -0,0 +1,96 @@
+package fr.ourten.brokkgui.wrapper;
+
+import org.lwjgl.opengl.GL11;
+
+import fr.ourten.brokkgui.BrokkGuiPlatform;
+import fr.ourten.brokkgui.internal.EGuiRenderMode;
+import fr.ourten.brokkgui.internal.IGuiHelper;
+import fr.ourten.brokkgui.internal.IGuiRenderer;
+import fr.ourten.brokkgui.paint.EGuiRenderPass;
+import net.minecraft.client.renderer.Tessellator;
+import net.minecraft.client.renderer.VertexBuffer;
+import net.minecraft.client.renderer.vertex.DefaultVertexFormats;
+
+public class GuiRenderer implements IGuiRenderer
+{
+    private final Tessellator  t;
+    private final VertexBuffer vertexBuffer; // Added
+    private final GuiHelper    helper;
+
+    public GuiRenderer(final Tessellator t)
+    {
+        this.t = t;
+        this.vertexBuffer = Tessellator.getInstance().getBuffer();
```

Comment:
Actually it would be more consistent to keep using and accessing the vertexBuffer from the tessellator instead of storing an instance of it.